zarr/storage.py

```diff
@@ -1883,3 +1883,121 @@ def __delitem__(self, key):
         with self._mutex:
             self._invalidate_keys()
             self._invalidate_value(key)
+
+
+class ChunkCache(MutableMapping):
+    """Storage class that implements a least-recently-used (LRU) cache for array chunks.
```

Comment:
Edit "Storage class that implements ..." to "Class that implements ..." or even just "A least-recently-used (LRU) cache ..." (as this isn't a storage class).